test/e2e/storage/utils/utils.go

```diff
@@ -619,6 +619,15 @@ func WaitForGVRDeletion(ctx context.Context, c dynamic.Interface, gvr schema.Gro
 	return fmt.Errorf("%s %s is not deleted within %v", gvr.Resource, objectName, timeout)
 }
 
+// EnsureGVRDeletion check for a certain period of a non-namespaced object has been deleted
+func EnsureGVRDeletion(ctx context.Context, c dynamic.Interface, gvr schema.GroupVersionResource, objectName string, poll, timeout time.Duration) error {
```

Comment:
Besides using `Eventually` when it should use `Consistently`, the description also is unclear.

```suggestion
// EnsureGVRDeletion checks that no object as defined by the group/version/kind and name is ever found during the given time period.
func EnsureGVRDeletion(ctx context.Context, c dynamic.Interface, gvr schema.GroupVersionResource, objectName string, poll, timeout time.Duration) error {
```

Reply:
Would it make sense to also support a namespace? If non-empty, use it, otherwise not. The function then becomes usable also for namespaced objects.

Reply:
yes, it does make sense.

Reply:
add namespace for both functions